mix.lock

```diff
@@ -1,19 +1,19 @@
 %{
-  "cowboy": {:hex, :cowboy, "2.6.1", "f2e06f757c337b3b311f9437e6e072b678fcd71545a7b2865bdaa154d078593f", [:rebar3], [{:cowlib, "~> 2.7.0", [hex: :cowlib, repo: "hexpm", optional: false]}, {:ranch, "~> 1.7.1", [hex: :ranch, repo: "hexpm", optional: false]}], "hexpm"},
```

Comment:
I'm not that familiar with this mix.lock format. Do you have a link to something that explains it? Does it change anything or is it purely a format change? How did you make this change? Do you know if it is backward compatible with older versions of elixir?

Reply:
> Do you have a link to something that explains it?

the mix.lock is not documented and is considered a private implementation detail of hex. I just confirmed this on the elixir slack in the #hex channel
![mix lock format](https://user-images.githubusercontent.com/9973/105531632-b6167000-5c8d-11eb-8140-7e415798d841.png)

> Does it change anything or is it purely a format change?

It's purely a format change

> How did you make this change?

This change was made "automatically", I think it was when I ran `mix deps.get`

> Do you know if it is backward compatible with older versions of elixir?

I believe that mix.lock changes are usually, but not always backwards compatible. But generally I don't believe this will be an issue in this case, because when someone is using master_proxy as a library (i.e. 99% of uses) the mix.lock file is not read. The mix.lock file is only read when someone is actually developing on master_proxy, and I think requiring a recent elixir version is fine in that case.